dotcom-rendering/src/lib/getAdPositions.ts

```diff
@@ -74,6 +75,34 @@ export const getMobileAdPositions = (
 		// Should insert no more than 10 ads
 		.slice(0, 10);
 
+/**
+ * Uses a very similar approach to pressed fronts, except we
+ * - Don't need to consider thrashers
+ * - Don't need to consider the 'most viewed' container
+ *
+ * The types are also slightly different, as we
+ */
+export const getTagFrontMobileAdPositions = (
+	collections: Array<GroupedTrailsBase>,
+	merchHighPosition: number,
+): number[] =>
+	collections
+		.filter(
+			(_, index) =>
+				!hasAdjacentCommercialContainer(index, merchHighPosition),
+		)
+		.filter(isEvenIndex)
+		.map((collection) =>
+			collections.findIndex(
+				(c) =>
+					c.day === collection.day &&
+					c.month === collection.month &&
+					c.year === collection.year,
```

Comment:
Destructuring? You know you want it…

```suggestion
				({ day, month, year }) =>
					day === collection.day &&
					month === collection.month &&
					year === collection.year,
```